frameworks/hdfs/src/main/dist/hdfs_svc.yml

```diff
@@ -166,24 +179,14 @@ plans:
         strategy: serial
         pod: name
         steps:
-          - pod-instance: 0
-            tasks: [format]
-          - pod-instance: 0
-            tasks: [node]
-          - pod-instance: 1
-            tasks: [bootstrap]
-          - pod-instance: 1
-            tasks: [node]
+          0: [format, node]
```

Comment:
This section is fully manual, with the keys being pod indexes.

Example:

```
- 0: [format-index, format-data]
- 1: [networking]
- 0: [server]
- 1: [server]
```

In this case, the above ordering will be followed to the letter: Pod 0 will run `format-index` and `format-data` in parallel, then Pod 1 will run `networking`, then Pod 0 will run `server, then Pod 1 will run `server` as well.